crates/key-value-redis/src/lib.rs

```diff
@@ -24,6 +25,7 @@ impl KeyValueRedis {
 
 #[async_trait]
 impl StoreManager for KeyValueRedis {
+    #[instrument(name = "spin_key_value_redis.get_store", skip(self), err(level = Level::INFO), fields(otel.kind = "client"))]
```

Comment:
A fret here is that the `get_store` and `get_keys` names have the same "shape" but are at very different scopes.  I am not sure if that's a problem though (and I sure don't have a better solution).

Reply:
Yeah I don't love it either but I've spent so long mulling over these names I think I just need to start merging things. We can come back and cleanup if inspiration strikes later 🤷